### PR TITLE
initial push for dashboard get and import

### PR DESCRIPTION
**Pull request by ianrscottexp**
New Features:
- Get Dashboard Information
- Import Dashboard
- Export Dashboard

### CODE REVIEW

**Review by mgaruccio — CHANGES_REQUESTED** [review]
Have a few notes on small updates, but also need to make sure that the pipfile and requirements files get updated so we can apply to tower.

File: galaxy.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 ---
-version: 2.1.2
+version: 2.3.2
```
[the remark this answers is not in the document]
Comment (ianrscottexp) [reply galaxy.yml:16]: Version is currently 2.3.1.  I should of rebased before I started to code

File: plugins/module_utils/kibana.py [hunk 2]
```diff
@@ -44,6 +44,32 @@ def send_api_request(self, endpoint, method, data=None):
       raise e ## This allows errors raised during the request to be inspected while debugging
     return loads(response.read())
 
+  def send_file_api_request(self, endpoint, method, data=None, file=None):
+    import requests
```
[the remark this answers is not in the document]
Comment (ianrscottexp) [reply plugins/module_utils/kibana.py]: I have moved the imports

**Review by mgaruccio — COMMENTED** [review]
lgtm

**Review by mgaruccio — APPROVED** [review]

**Review by exp-hc — APPROVED** [review]
LGTM

**Pull request merged**